sapl/sessao/views.py

```diff
@@ -561,7 +561,12 @@ def get_presencas(self):
         )
         presentes = [p.parlamentar for p in presencas]
 
-        for parlamentar in Parlamentar.objects.filter(ativo=True):
+        mandato = Mandato.objects.filter(
+            legislatura_id=self.object.legislatura_id)
+
+        mandato_parlamentar = [p.parlamentar for p in mandato]
+
+        for parlamentar in mandato_parlamentar:
             if parlamentar in presentes:
```

Comment:
@eduardocalil , entendi. Mas como fica a passagem do parâmetro nos arquivos sapl/templates/sessao/presenca.html e presenca_ordemdia.html ?
Especificamente linha 22:
   {% for parlamentar, check in view.get_presencas %} ou
   {% for parlamentar, check in view.get_presencas_ordem %}